packages/domain-picker/src/components/index.tsx

```diff
@@ -119,6 +119,9 @@ export interface Props {
 
 	/** Vendor string for domain suggestions */
 	vendor?: string;
+
+	/** Hides the recommendation label */
+	hideRecommendationLabel?: boolean;
```

Comment:
Thanks @razvanpapadopol  that is indeed more readable :)
It's something that I do follow but since the default behaviour was to show the label I went with the negative form of the prop!